File: Domovoy/Models/Informer/EventInformer.cs
namespace Domovoy.Models;

public class EventInformer
{
    public int Id { get; set; }
    public Informer Informer { get; set; }
    public int InformertId { get; set; }

    /// <summary>
    /// Активен прямо сейчас
    /// </summary>
    public bool IsNowActive { get; set; }
    public List<ActiveSession> ActiveSessions { get; set; }
    public DateTime? Time { get; set; }
}

File: Domovoy/ClientApp/src/api/models/EventInformer.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ActiveSession } from './ActiveSession';
import type { Informer } from './Informer';

export type EventInformer = {
    id?: number;
    informer?: Informer;
    informertId?: number;
    /**
     * Активен прямо сейчас
     */
    isNowActive?: boolean;
    activeSessions?: Array<ActiveSession> | null;
    time?: string | null;
};